content/python/concepts/built-in-functions/terms/print/print.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'print()'
+Description: 'Outputs the string representation of an object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `print()` function takes in a string parameter or any other element, such as an integer. The parameter is converted to a string through an implicit call of the built-in `str()` function and the value is outputted.
+
+## Syntax
+
+```py
+print(object)
+```
+
+## Example
+
+```py
+print("Hello, World")
+```
+
+The output would be:
+
+```pseudo
+Hello, World!
+```
+
+**Note:** To print an integer of a floating-point number, use the str() function.
```

Comment:
Can we expand on this with, perhaps, a codebyte/py example?

Reply:
@KabirCampwala @Dusch4593 I'm actually a bit confused by this note. When we print an integer or floating-point in Python, can't we just do (for example) `print(3)` or `print(3.14)` which would output 3 and 3.14, respectively?

Reply:
True. Plus, we already have an entry for the `str()` function underway. Gonna remove this and include a codebyte example about using the `sep` and `end` parameters.